stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  required: true
+  default: ""
+  type: string
+  label: Swift Password
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.swift.container
+  required: true
+  default: ""
+  type: string
+  label: Swift Storage Container Name
+  show_if: "persistence.imageChartStorage.type=swift"
+  group: "OCI Artifact Storage Settings"
+# registry config
+- variable: persistence.persistentVolumeClaim.registry.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Registry
+  show_if: "persistence.enabled=true&&persistence.imageChartStorage.type=filesystem"
+  group: "Registry Service Settings"
+- variable: persistence.persistentVolumeClaim.registry.size
+  required: true
+  default: "5Gi"
+  description: "Registry Persistent Volume Size"
+  type: string
+  label: Registry Volume Size
+  group: "Registry Service Settings"
+  show_if: "persistence.enabled=true&&persistence.imageChartStorage.type=filesystem"
+- variable: persistence.persistentVolumeClaim.registry.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Registry
+  group: "Registry Service Settings"
+  show_if: "persistence.enabled=true&&persistence.imageChartStorage.type=filesystem"
+# jobservice config
+- variable: jobservice.jobLogger
+  default: "database"
+  required: true
+  description: "Config JobService JobLogger type."
+  type: enum
+  label: Config JobService JobLogger Type
+  group: "Jobservice Settings"
+  options:
+  - "file"
+  - "database"
+  - "stdout"
+- variable: persistence.persistentVolumeClaim.jobservice.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Jobservice
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+- variable: persistence.persistentVolumeClaim.jobservice.size
+  required: true
+  default: "1Gi"
+  description: "Jobservice Persistent Volume Size"
+  type: string
+  label: Jobservice Volume Size
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+- variable: persistence.persistentVolumeClaim.jobservice.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Jobservice
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+# database
+- variable: database.type
+  required: true
+  default: "internal"
+  description: "Config Database type, set to internal to deploy a database server as part of this deployment."
```

Comment:
What about using full "Configure" in description?

Reply:
Fixed